### ISSUE TITLE
`Programming Exercises`: Spacing for online/offline editor mode weird in exercise list for instructor

### ISSUE THREAD

**Issue by b-fein** [ls1intum/Artemis#9302]
### Describe the bug

See screenshot. The spacing between the left part, the colon, and the right part does not follow usual text layouts.

### To Reproduce

As instructor, go to the exercise overview for a course. Look at the list of programming exercises.

### Expected behavior

The text looks like
```
IDE: Yes
Online: Yes
Online IDE: No 
```
or maybe
```
IDE:        Yes
Online:     Yes
Online IDE:  No 
```
but with the colon definitely being part of the left side.

### Screenshots

![image](https://github.com/user-attachments/assets/1fc795e0-fc1d-4cac-9592-96c2f7fc6500)
![image](https://github.com/user-attachments/assets/815c9a29-6dfa-4bf4-ba64-225c4053c66b)

### Which version of Artemis are you seeing the problem on?

7.5.1

### What browsers are you seeing the problem on?

Firefox

**Comment by krusche** [comment]
@iyannsch can we only display the OnlineIDE state if the `theia` profile is active?

**Comment by iyannsch** [comment]
I think both of your concerns are already addressed in [#9293](https://github.com/ls1intum/Artemis/pull/9293)

**Comment by b-fein** [comment]
Is fixed in newer Artemis versions.
